open_academy/models/course.py

```diff
@@ -1,4 +1,4 @@
-from odoo import models, fields
+from odoo import models, fields, api
 
 
 class Course(models.Model):
```

Comment:
Forgot said this, but if you add `rec_name` to the model can use the title as record.name

```suggestion
class Course(models.Model):
    _name = 'course'
    _rec_name = 'title'
```